### PR TITLE
EIP-2633: Formal Proposal for Upgradable Governance

### PR DESCRIPTION
**Pull request by edsonayllon**
Creates eip-2633.

### CODE REVIEW

File: EIPS/eip-2633.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: Formal Proposal for Upgradable Governance
+author: Edson Ayllon (@edsonayllon)
+discussions-to: https://ethereum-magicians.org/t/eip-2633-formal-proposal-for-upgradable-governance/4264
+status: Draft
+type: Meta
+created: 2020-05-06
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:12]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:15]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+A formal proposal to set a precedent that Ethereum's governance structure can be changed and improved over time. 
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:20]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+In a survey done collecting feedback from EIP Editors, EIP Authors, and Core Developers, a need was signaled for a formal decision-making process for network upgrade EIP inclusion. While a formal decision-making process can be created, a proper governance solution may take years of research and funding for that research. This proposal is to allow to begin to create a formal decision-making process, while not committing to that formal decision-making process indefinitely, and encourage more improvements to decision making to be pursued and possibly funded.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:25]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+This proposal is to set a precedent for an upgradable governance system for Ethereum. The concept is that Ethereum's governance is not fixed, and can be improved on either regularly or as the need arises. Here, governance is defined as decision making. Currently, this would apply to network upgrade coordination (hard-forks), and not the EIP drafting process itself. 
+
+The initiating of an upgrade process can be done through motioning or by a schedule for example. The process of initiating a governance upgrade is unspecified in this EIP.
+
+The process of how improvements are made is also unspecified.
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:34]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+How an upgrade process is initiated is kept out of the scope of this EIP, as what can trigger the upgrade process itself can be improved over time. If a formal upgrade initiation process is specified, it may be specified as an EIP which may also be superseded over time. 
+
+The process of how improvements are made is also unspecified, as that process can also be improved over time, and maybe different for the needs of each governance upgrade. 
+
+## Backwards Compatibility
+<!--All EIPs that introduce backward incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backward compatibility treatise may be rejected outright.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:41]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+Currently, Ethereum's governance is done off-chain. No compatibility issues.
+
+## Test Cases
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:46]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+No test cases.
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:51]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+No implementation yet. 
+
+## Security Considerations
+<!--All EIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surface risks, and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks, and how they are being addressed. EIP submissions missing the "Security Considerations" section will be rejected. An EIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.-->
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:56]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+If changes to governance are poorly made, there would be large security concerns in terms of a single party swaying decisions. 
+
+## Copyright
+
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:60]:
```suggestion
```

File: EIPS/eip-2633.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+This proposal is to set a precedent for an upgradable governance system for Ethereum. The concept is that Ethereum's governance is not fixed, and can be improved on either regularly or as the need arises. Here, governance is defined as decision making. Currently, this would apply to network upgrade coordination (hard-forks), and not the EIP drafting process itself. 
+
+The initiating of an upgrade process can be done through motioning or by a schedule for example. The process of initiating a governance upgrade is unspecified in this EIP.
+
+The process of how improvements are made is also unspecified.
```
Comment (MicahZoltu) [inline EIPS/eip-2633.md:30]: EIPs are explicitly specifications, and this EIP doesn't specify anything so it isn't a valid EIP.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
This doesn't appear to actually specify anything, and EIPs are explicitly specifications for things.  I recommend either enhancing this to be a specification for a system of governance, or closing this and continuing this discussion over at https://ethereum-magicians.org/.

It is also worth noting that EIPs are unrelated to the Ethereum governance (hard fork inclusion process).  Hard fork inclusion is now handled over at https://github.com/ethereum/eth1.0-specs/

File: EIPS/eip-2633.md (hunk 11 above)
Comment (edsonayllon) [reply EIPS/eip-2633.md]: Greg said the same thing a while back. I included specification with c48749c

**Comment by MicahZoltu** [comment]
I'm generally pretty against using EIPs repository for anything related to the Ethereum hard fork process.  IMO, the EIPs repository is for technical specifications only.  You may want to try proposing something over at https://github.com/ethereum/eth1.0-specs/, which is where the hardfork decision making will happen, though historically the Ethereum community is pretty resistant to formalized processes (for good reason I think), so I suspect you'll get resistance anywhere you try to push for formalizing governance.

**Pull request closed without merging**